tests/FakeItEasy.Specs/FakeItEasy.Specs.ruleset

```diff
@@ -5,4 +5,4 @@
     <Rule Id="CA1003" Action="None" />
     <Rule Id="CA1009" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
Yes, I must have opened it in text view. Anyway, having the empty line at the end is correct according to the .editorconfig, right?